KLR/Util.lean

```diff
@@ -46,3 +46,9 @@ notation f "▷" l =>
 
 def impossible {a : Type} [h : Inhabited a] (msg : String := "") :=
   @panic a h s!"Invariant violation: {msg}"
+
+def get! [Inhabited a] (x : Err a) : a := match x with
```

Comment:
We can get these (and impossible) from TensorLib when we link it, but for now I thought copying was better.